data/implementations/discovery.json

```
@@ -59,6 +69,11 @@
           "name": "Go",
           "status": "Done",
           "url": "https://github.com/libp2p/go-libp2p/blob/master/p2p/discovery/mdns.go"
+        },
+        {
+          "name": "Rust",
+          "status": "Unstable",
```

```suggestion
          "status": "Usable",
```

We use it within Substrate. I have not checked whether it is compatible with the Golang implementation.

---

Unstable is orange ("prototype") and Usable is yellow ("in progress") -- so seems reasonable to me!